lib/ui/semantics.dart

```diff
@@ -618,9 +618,13 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
   /// string describes what result an action performed on this node has. The
   /// reading direction of all these strings is given by `textDirection`.
   ///
-  /// The fields 'textSelectionBase' and 'textSelectionExtent' describe the
```

Comment:
This needs to be added to the web_ui implementation too

Reply:
Done.